packages/bpk-component-definitions-list/src/BpkDefinition.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './bpk-definitions-list.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkDefinition = (props) => {
+  const classNames = [getClassName('bpk-definitions-list__definition')];
+
+  if (props.className) { classNames.push(props.className); }
+
+  return (<dd className={classNames.join(' ')}>{props.children}</dd>);
```

Comment:
The parenthesis are redundant here right? Can we remove them?

Reply:
this should spread arbitrary props like {...rest}

Reply:
will do - btw this was lifted from `bpk-component-list` so may want to update there as well